app/src/main/java/com/practicum/playlistmaker/media/ui/activity/CurrentPlaylistFragment.kt

```diff
@@ -0,0 +1,313 @@
+            override fun onStateChanged(bottomSheet: View, newState: Int) {
+                when (newState) {
+                    BottomSheetBehavior.STATE_HIDDEN -> {
+                        binding.overlay.visibility = View.GONE
+                    }
+
+                    else -> {
+                        binding.overlay.visibility = View.VISIBLE
+                    }
+                }
+            }
+
+            override fun onSlide(bottomSheet: View, slideOffset: Float) {
+                binding.overlay.alpha = slideOffset
+            }
+        })
+
+
+        bottomSheetMenu.addBottomSheetCallback(object :
+            BottomSheetBehavior.BottomSheetCallback() {
+            override fun onStateChanged(bottomSheet: View, newState: Int) {
+                when (newState) {
+                    BottomSheetBehavior.STATE_HIDDEN -> {
+                        binding.overlay.visibility = View.GONE
+                    }
+
+                    else -> {
+                        binding.overlay.visibility = View.VISIBLE
+                    }
+                }
+            }
+
+            override fun onSlide(bottomSheet: View, slideOffset: Float) {
+                binding.overlay.alpha = slideOffset
+            }
+        })
+    }
+
+    override fun onStart() {
+        super.onStart()
+        viewModel.getPL(id)
+    }
+
+    private fun render(state: TrackListState) {
+        when (state) {
+            is TrackListState.TrackListContent -> {
+                pl = state.playlist
+                tracks = state.tracks.reversed()
+                showContent(
+                    tracks,
+                    state.playlist,
+                    state.totalTime
+                )
+            }
+
+            is TrackListState.NoContent -> {
+                pl = state.playlist
+                Toast.makeText(
+                    requireContext(), "В этом плейлисте не треков",
+                    Toast.LENGTH_LONG
+                ).show()
+                showEmpty(state.playlist, state.totalTime)
+            }
+        }
+    }
+
+    private fun showEmpty(playlist: Playlist, time: String) {
+        binding.bottomSheet.visibility = View.GONE
+        val uri = getImage(playlist)
+
+        Glide.with(requireContext())
+            .load(uri)
+            .placeholder(R.drawable.placeholder)
+            .centerCrop()
+            .transform(RoundedCorners(8))
+            .into(binding.coverImageView)
+
+        binding.PLNameTextView.text = playlist.playlistName
+        binding.descriptionTextView.text = playlist.playlistdescription
+
+        val minutePlural = this.resources.getQuantityString(plurals_3, time.toInt(), time.toInt())
+        binding.PLDurationTextView.text = minutePlural
+
+        val count = playlist.tracksCount
+        val trackPlural = this.resources.getQuantityString(plurals_2, count, count)
+        binding.trackCountTextView.text = "$trackPlural"
```

Comment:
Здесь вроде можно просто
```suggestion
        binding.trackCountTextView.text = trackPlural
```